src/expandable-block/block.scss

```diff
@@ -149,4 +149,16 @@
       }
     }
   }
+
+  &.iui-small .iui-header {
+    padding: $iui-baseline * 0.5 $iui-s;
+
+    > .iui-expandable-block-label {
+      margin-left: $iui-s;
+    }
+
+    > .iui-status-icon {
```

Comment:
With 12px size icons:
![image](https://user-images.githubusercontent.com/36186912/136564719-3acb6255-354d-4b16-8a9f-11cabf99fce4.png)
Status icon feels a bit too small here, expand icon is ok.

Reply:
Which one? 😀